proxy/logging/LogFilter.h

```diff
@@ -388,30 +388,33 @@ LogFilterString::_checkCondition(OperatorFunction f, const char *field_value, si
 
 --------------------------------------------------------------------------*/
 static void
-wipeField(char **dest, char *field)
+wipeField(char **dest, char *field, const char *uppercase_dest)
```

Comment:
It's much easier to read because uppercase_dest has const qualifier. However, why does it have "_dest" suffix? It's no longer a destination.

Reply:
Updated.